Sources/Alert/AlertStyle.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+extension Warp {
+    /// <#Description#>
+    public enum AlertStyle: Equatable {
+        /// <#Description#>
```

Comment:
I would copy and paste the descriptions from Figma to here.
<img width="389" alt="Screenshot 2024-01-26 at 12 29 49" src="https://github.com/warp-ds/warp-ios/assets/89784012/76353c2c-b25b-49f7-b60c-efc948b86fbd">

Reply:
Fantastic, thanks!

Reply:
Can I ask where it is in Figma?
I can't find it in [Alert](https://www.figma.com/file/8P1JQsd82b93gQ6K3igO2p/Warp---Components?type=design&node-id=171-20423&mode=design&t=PvP14KlqRUqXVjRw-0) section!